### ISSUE TITLE
Latest homer webapp failed to return 200 in CI

### ISSUE THREAD

**Issue by dougbtv** [sipcapture/homer5-docker#7]
Ooops! So, was experimenting with getting the latest Homer running with this suite of docker images, etc, and... Noticed that CI failed with it.

Couple things here that I'll address...
- Get it to work with current version
- Possibly instead of pulling head on master from Homer, pull a specific tag (still have to look into this)

Hopefully look at it later tonight (4/22) or hopefully this weekend.

**Comment by lmangani** [comment]
I'll be around to spare a hand, we could also try use the new rotation scripts to keep all the credentials for the other components ie: https://github.com/sipcapture/homer-api/tree/master/scripts/new

**Comment by dougbtv** [comment]
Excellent! I'll give the new rotation scripts a whirl for sure.

**Comment by dougbtv** [comment]
I gave the new rotation scripts a whirl, but, I think it might need a little grooming before I implement it, at least I'm running into a few issues, I fixed one issue and [submitted this PR (for fixing ini location)](https://github.com/sipcapture/homer-api/pull/82). However I also ran into a [MySQL issue that I didn't dig into, but reported it in this issue](https://github.com/sipcapture/homer-api/issues/81).

Otherwise I _love_ the new rotation scripts, they're looking good, and the config file construct is really nice.

I'm going to put my changes I had while working on it in a separate branch and push it up, I'll call it "new-rotation" or something like that. And I'll come back to it later.

Getting closer to getting this guy back into health though, I'll update once I've got some measurable progress.

**Comment by dougbtv** [comment]
I found it, turns out my modified way of copying the rotation scripts was incorrect. It failed to copy all of the files, so! That made it so the new tables for the day weren't created when initially bootstrapping the rotation scripts. 

Aaaand, the title of this issue is wrong -- it wasn't the webapp that failed I read the CI report wrong -- it was that it didn't see the expected results in the database (as the homer_data.\* table that needed to be there for today wasn't there). 

I've got that fixed, waiting for the CI to finish and I'll merge it into master.
